scripts/k8s/deploy_kubeflow.sh

```diff
@@ -135,73 +121,37 @@ function install_dependencies() {
   fi
 }
 
+function clone_repo() {
+  pushd .
+  mkdir -p "${KUBEFLOW_MANIFESTS_DEST}"
+  pushd "${KUBEFLOW_MANIFESTS_DEST}"
+  git clone -b "${KUBEFLOW_MANIFESTS_VERSION}" "${KUBEFLOW_MANIFESTS_URL}" .
 
-function install_mpi_operator() {
+  # Inject custom dex config
+  cp -v "${KUBEFLOW_DEEPOPS_DEX_CONFIG}" "${KUBEFLOW_MANIFESTS_DEST}/common/dex/base/config-map.yaml"
+  cp -v "${KUBEFLOW_DEEPOPS_USERNS_PARAMS}" "${KUBEFLOW_MANIFESTS_DEST}/common/user-namespace/base/params.env"
 
-  # Download kustomize, as required by mpi
-  pushd .
-  cd ${CONFIG_DIR}
-  curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases |\
-    grep browser_download |\
-    grep linux |\
-    cut -d '"' -f 4 |\
-    grep /kustomize/v |\
-    sort | tail -n 1 |\
-    xargs curl -s -O -L
-  tar xzf ./kustomize_v*_linux_*.tar.gz
-  mv kustomize ${KUSTOMIZE}
-
-  mkdir -p ${KUBEFLOW_MPI_DIR}
```

Comment:
Does the current install include the MPI Operator?

Reply:
It's worth noting that the latest version of MPI Operator changed the control mechanism and multi-node workloads will no longer run without ssh capabilities in the container.